File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1039.c
#include <stdio.h>
#include <math.h>

int main(){
    double r1, x1, y1, r2, x2, y2;

    while(scanf("%lf %lf %lf %lf %lf %lf", &r1, &x1, &y1, &r2, &x2, &y2) != EOF){

        double d = (x1-x2)*(x1-x2) + (y1-y2)*(y1-y2);

        if(d <= r1*r1 && sqrt(d)+r2 <= r1) printf("RICO\n");
        else printf("MORTO\n");
    }
    
    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1039.cs
using System;

class URI
{
    static void Main() {
        while(true) {
            string linha = Console.ReadLine();
            if (string.IsNullOrEmpty(linha))
                break;
            string[] ls = linha.Split(' ');

            double r1 = double.Parse(ls[0]), x1 = double.Parse(ls[1]), y1 = double.Parse(ls[2]), r2 = double.Parse(ls[3]), x2 = double.Parse(ls[4]), y2 = double.Parse(ls[5]);
            
            double d = (x1-x2)*(x1-x2) + (y1-y2)*(y1-y2);

            if(d <= r1*r1 && Math.Sqrt(d)+r2 <= r1) Console.Write("RICO\n");
            else Console.Write("MORTO\n");

        }
    }
}
